shared/searchv3/results-list/container.js

```diff
@@ -5,8 +5,13 @@ import SearchResultsList from '.'
 import type {TypedState} from '../../constants/reducer'
 
 // TODO use entities
-const mapStateToProps = (state: TypedState, id: string) => {
-  return {}
+const mapStateToProps = ({entities}: TypedState, {keyPath}) => {
+  // $FlowIssue
+  const items = entities.getIn(keyPath).keySeq().toArray()
```

Comment:
we don't do that with our old search code, and it looks like we do get a sorted list from the api